src/plugins/d3/renderer/hooks/useSeries/types.ts

```diff
@@ -69,6 +75,7 @@ type BasePreparedSeries = {
     color: string;
     name: string;
     id: string;
+    index: number;
```

Comment:
Maybe it's not worth storing the index? Colors, for example, can also be distributed automatically, but you don't need to store an index for this